src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
+
+    virt_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Virtualization type """
+
+    accounts = attr.ib(type=str, default=None, validator=optional_str)
+    """Accounts """
+
+    groups = attr.ib(type=str, default=None, validator=optional_str)
+    """Groups"""
+
+    release_notes = attr.ib(type=str, default=None, validator=optional_str)
+    """"Release notes for AMI """
+
+    operating_system_name = attr.ib(type=str, default=None, validator=optional_str)
+    """Name of operating system"""
+
+    operating_system_version = attr.ib(type=str, default=None, validator=optional_str)
+    """Version of operating system"""
+
+    usage_instructions = attr.ib(type=str, default=None, validator=optional_str)
+    """Instructions for use of AMI"""
+
+    recommended_instance_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Recommended instance type for AMI"""
+
+    product_type = attr.ib(type=str, default=None, validator=optional_str)
+    """Type of product"""
```

Comment:
What is expected to be here? If we have some examples it would be better.

Reply:
product_type would be AMIProduct, SaasProduct, ServerProduct (which is also AMIProduct, I couldn't differentiate the two.)